lisa/runner.py

```diff
@@ -266,14 +275,14 @@ def _start_loop(self) -> None:
             task_manager = TaskManager[List[TestResult]](
```

Comment:
The `if self._runners:` can be changed to below. It can reduce complexity of this method.
```python
if not self._runners:
    return
```